### PR TITLE
Rules Updates for multiple Nodes

### PR DESCRIPTION
**Pull request by liorkhe**
Rules Updates for multiple Nodes

### CODE REVIEW

File: inference_rules.adoc [hunk 1]
```diff
@@ -468,19 +470,19 @@ Examples of forbidden bus interconnect include: PCIe/CXL/CCIX, Hypertransport, N
 
 Additionally, any interconnect not listed in the permitted list is forbidden unless clearance is first obtained from the MLPerf Inference WG.
 
-The SUT and QDL must run on two physically separated and distinct systems.
+The SUT and QDL must run on two physically separated and distinct systems. SUT can contain multiple Nodes.
```
Comment (saareliad) [inline inference_rules.adoc]: The SUT

File: inference_rules.adoc [hunk 2]
```diff
@@ -445,8 +445,10 @@ Non-conforming network submission should be submitted to Open category, under th
 * The QDL is not allowed to pad the data in queries.
 * The QDL is not allowed to cache queries or responses.
 * The QDL is implementing the network function of the LoadGen Node towards the SUT node and handles the required processing. E.G. padding of the payload as required by the network protocol.
+* The QDL should reflect a single SUT to the LoadGen. LoadGen Operates with a single SUT. 
 * The Name method's return value must contain the substring "Network SUT".
 * The Name method's implementation must include at least one round trip over the network. The Name method must not return until the round trip is complete.
+* The QDL should serially get the Name from all the Nodes of the SUT, aggregate the response in the Name Method after getting response from all Nodes. Each Node must have a unique Name.
```
Comment (saareliad) [inline inference_rules.adoc]: get the names

File: inference_rules.adoc (hunk 1 above)
Comment (liorkhe) [reply inference_rules.adoc]: thx

File: inference_rules.adoc (hunk 2 above)
Comment (liorkhe) [reply inference_rules.adoc]: thx

**Pull request merged**